.travis.yml

```diff
@@ -40,7 +40,10 @@ before_script:
   - grunt nsp --force
 
 script:
-  - npm test
+  # 'npm run test-travis' runs all tests together to collect all coverage into one from mysql and db-server
+  - COVERALLS_REPO_TOKEN=mlOESL8slkePkodJpcvhF1AUKUH3llXi8 npm run test-travis
+  # memory tests are separate to avoid being included into coverage because they run through the same code
+  - npm run test-mem
```

Comment:
I wonder if it makes sense to ignore coverage from memory db? Seems like it could be useful since developing/testing locally I mostly use memory db.